vanilla-framework/vanilla-framework
# Correct box shadow and font-size on Tabs pattern

## Done

- Update box shadow on `.p-card--highlight`
- Update default font-size of tabs

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/patterns/tabs/
- Check box shadow is `0 1px 5px 0`
- Check font-size of tabs is 14px

## Details

Fixes #1613, Fixes #1614
